### PR TITLE
[CDAP-11666][CDAP-11768] Icons + Market/preview bugfixes

### PR DESCRIPTION
**Pull request by tonybach**
Added new icons, and added Azure and AWS market tabs. 

Also fixed these JIRAs:
- https://issues.cask.co/browse/CDAP-11666?focusedCommentId=30069&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-30069
- https://issues.cask.co/browse/CDAP-11768

### CODE REVIEW

File: cdap-ui/app/cdap/styles/fonts/symbol-defs.svg [hunk 1]
```diff
@@ -3143,11 +3143,11 @@
...
 </symbol>
 <symbol id="icon-lock_open" viewBox="0 0 32 32">
 <title>lock_open</title>
-<path d="M26.7 11.8c-0.6-0.6-1.3-0.8-2-0.8h-1.4v-3c0-2-0.7-3.7-2.1-5.1s-3.1-2.1-5.1-2.1-3.7 0.7-5.1 2.1c-0.3 0.3-0.6 0.7-0.9 1.1-0.5 0.7-0.3 1.7 0.5 2.1v0c0.6 0.3 1.4 0.1 1.7-0.5 0.2-0.2 0.4-0.5 0.6-0.7 0.9-0.9 1.9-1.3 3.2-1.3s2.3 0.4 3.2 1.3c0.9 0.9 1.3 1.9 1.3 3.2v2.9h-13.2c-0.8 0-1.5 0.3-2 0.8s-0.9 1.2-0.9 2v14.5c0 0.8 0.3 1.4 0.9 2s1.3 0.8 2 0.8h17.4c0.8 0 1.5-0.3 2-0.8 0.6-0.6 0.9-1.2 0.9-2v-14.5c-0.1-0.7-0.4-1.4-1-2zM24.7 28.4h-17.4v-14.5h17.4v14.5z"></path>
+<path d="M5.8 13c-0.9 0-1.7 1-1.7 2.1v14.2c0 1.2 0.8 2.2 1.7 2.2h20.4c0.9 0 1.7-1 1.7-2.2v-14.1c0-1.2-0.8-2.1-1.7-2.1h-0.6v-2.8c0-5.2-4.1-9.6-9.3-9.7-0.1 0-0.4 0-0.6 0-2.4 0-4.5 1-6.1 2.5-0.9 0.8-0.8 2.2 0.1 3v0c0.7 0.6 1.9 0.7 2.6 0 1.2-1.1 2.8-1.6 4.6-1.4 2.8 0.4 4.8 3 4.8 5.8v2.6h-15.9zM17.9 22.2v4.3c0 0.5-0.4 0.9-0.9 0.9h-2c-0.5 0-0.9-0.4-0.9-0.9v-4.3c-0.5-0.5-0.8-1.1-0.8-1.8 0-1.4 1.1-2.5 2.4-2.6 0.1 0 0.4 0 0.6 0 1.3 0.1 2.4 1.2 2.4 2.6 0 0.7-0.3 1.4-0.8 1.8z"></path>
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/styles/fonts/symbol-defs.svg:3150]: This seems to have removed the lock icon from the header
screen shot 2017-06-01 at 6 43 34 pm

Comment (tonybach) [reply cdap-ui/app/cdap/styles/fonts/symbol-defs.svg:3150]: Doesn't seem like there was a lock icon at all for Local Sandbox. You can check release/4.2

Comment (ajainarayanan) [reply cdap-ui/app/cdap/styles/fonts/symbol-defs.svg:3150]: @bdmogal I remember you adding it in your PR. Can you confirm?

Comment (bdmogal) [reply cdap-ui/app/cdap/styles/fonts/symbol-defs.svg:3150]: the security icon? It shows if ``security.enabled = true``

Comment (bdmogal) [reply cdap-ui/app/cdap/styles/fonts/symbol-defs.svg:3150]: point it to a cluster and see if it shows up

Comment (ajainarayanan) [reply cdap-ui/app/cdap/styles/fonts/symbol-defs.svg:3150]: Verified the lock icon. 👍 
screen shot 2017-06-01 at 11 38 04 pm

**Review by ajainarayanan — APPROVED** [review]
Code LGTM 👍

**Comment by leacd** [comment]
LGTM

**Comment by tonybach** [comment]
Build: https://builds.cask.co/browse/CDAP-DRC6735-3

**Pull request merged**